mule-extensions-api-persistence/src/test/resources/extension/list-of-serialized-extension-model.json

```diff
@@ -898,7 +898,7 @@
             "minOccurs": 1,
             "childComponents": [
               {
-                "isRequired": false,
+                "isRequired": true,
```

Comment:
why?

Reply:
changed chains to be required by default, since chains are implicit (that is, without a wrapper element) and what required means to us is not a chain with one or more elements (that is a parameter size restriction) but wether or not the chain exists (even if it is empty)